### PR TITLE
fix: parse github emoji

### PR DESCRIPTION
**Pull request by atymic**
This PR adds a parser to convert the github emoji shortcodes into image tags (and some styles to size/align them).

I also remove the `index.md` docs file, as it's pulled in from the main repo, and should not have been committed before.
Built css/js removed as well.

### CODE REVIEW

File: source/_assets/sass/_documentation.scss [hunk 1]
```diff
@@ -1 +1,4 @@
-// Add your custom styles here
+img.gh-emoji {
```
Comment (jbelien) [inline source/_assets/sass/_documentation.scss]: The style doesn't seem to be "applied" on the [preview](https://deploy-preview-19--geocoder-php.netlify.com/docs/#local) and thus the flags are too big IMHO but maybe it's an just and issue with the preview.

Comment (atymic) [reply source/_assets/sass/_documentation.scss]: Strange, it's working on my dev. Will have another look tomorrow.

Comment (atymic) [reply source/_assets/sass/_documentation.scss]: image

Comment (atymic) [reply source/_assets/sass/_documentation.scss]: For some reason, it's not in the CSS that's being compiled 🤔 
https://deploy-preview-19--geocoder-php.netlify.com/assets/build/css/main.css

Comment (atymic) [reply source/_assets/sass/_documentation.scss]: Ok, fixed now :) Needed to add it to the purgecss whitelist.

Comment (jbelien) [reply source/_assets/sass/_documentation.scss]: Fixed indeed 👍

**Pull request merged**